js/holidays.js

```diff
@@ -354,51 +354,76 @@ function getMonthDays( year ) {
 }
 
 /**
- * Calculates the observation date for a given holiday according to a country's rules
+ * Returns an array of actual and observed holiday dates according to a country's rules
  *
- * @param {number} year
- * @param {number} month
- * @param {number} day
- * @param {string} country
+ * I.e. if a holiday falls on a Sunday and an Monday becomes a public holiday instead,
+ * both Sunday and Monday dates will be returned.
  *
- * @returns {string} holiday date in 'month-day' format
+ * @param {number} year Holiday year
+ * @param {number} month Holiday month
+ * @param {number} day Holiday day
+ * @param {string} country Country code
+ * @param {string} name Holiday name
+ * @param {Object} options Additional options
+ * @param {string} options.observedName Holiday name for the observed date
+ * @param {number} options.consecutive How many consecutive holidays is this one a part of (to avoid shifting observed date onto them)
```

Comment:
I don't think these should be additional `@param` lines, since they're part of the `options` object. Also, maybe we won't need the `observedName` option, since we're not using it anywhere (see the change I propose in the next comment).

Reply:
I went with https://jsdoc.app/tags-param.html#parameters-with-properties found via https://stackoverflow.com/a/31573441/288201.

The options object itself was added to specify multiple parameters with defaults - we can do without it for just one (`consecutive`) but if anything else is added in the future it will need a refactor.

Reply:
> I went with https://jsdoc.app/tags-param.html#parameters-with-properties found via https://stackoverflow.com/a/31573441/288201.

Ah ok, I stand corrected then.